### ISSUE TITLE
Can't compile

### ISSUE THREAD

**Issue by IzuiSP** [jason-e/rngfix#6]
plugin.sp(851) : error 017: undefined symbol "TR_EnumerateEntitiesHull"
plugin.sp(929) : error 017: undefined symbol "TR_EnumerateEntitiesHull"
plugin.sp(1209) : error 017: undefined symbol "TR_ClipCurrentRayToEntity"

**Comment by jason-e** [comment]
That's right, you must be using SourceMod 1.10 build 6326 or newer, which adds new traceray natives to SDKTools.
